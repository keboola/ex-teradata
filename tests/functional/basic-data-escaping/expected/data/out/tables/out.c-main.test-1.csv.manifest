{
  "incremental": false,
  "primary_key": []
}
