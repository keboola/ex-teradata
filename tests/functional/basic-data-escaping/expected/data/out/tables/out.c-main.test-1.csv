"column1","column2"
"unicode characters","ľš čť žý áí éú äô ň"
"columns with
new line","columns with 	tab"
"unicode initcap","Ľš Čť Žý Áí Éú Äô Ň"
"column with backslash \ inside","column with backslash and enclosure \"""""
"column with 
 	 \","second col"
"line with enclosure","second column"
"first","something with

double new line"
"column with enclosure """", and comma inside text","second column enclosure in text """""
